README.md

```diff
@@ -1 +1 @@
-# my-codereview
\ No newline at end of file
+# Ariane_Godfrey_Project
```

Comment:
It seems you haven't set up your linters. To do that, create a .github folder, add a workflows sub-folder then create a file linters.yml inside it. Copy the code from the linters file from this link, https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml and paste it into your repo's linter.
Once completed, create a .gitignore file, and inside it type node_modules. Go ahead and copy the .hintrc file and the .stylelintrc.json file from this link, https://github.com/microverseinc/linters-config/tree/master/html-css into your repo. Copy the code inside the files to yours as you did when creating linter.yml but do not create any folder this time. Your linters should be good to go. 👍